  0%|          | 0/3 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 3/3 [00:00<00:00, 32430.19it/s]
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 63.84it/s]
  0%|          | 0/3 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 3/3 [00:00<00:00, 58254.22it/s]
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 75.19it/s]
  0%|          | 0/3 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 3/3 [00:00<00:00, 79638.68it/s]
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 62.10it/s]
405 GET /corrector (106.55.202.206) 0.40ms
404 GET /correctorHSavePOSThttp:/ (157.148.40.30) 0.41ms
  0%|          | 0/3 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 3/3 [00:00<00:00, 55924.05it/s]
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 54.69it/s]
  0%|          | 0/4 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 4/4 [00:00<00:00, 102300.10it/s]
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 40.73it/s]
404 GET /correctorHSave (106.55.200.29) 0.41ms
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 39016.78it/s]
  0%|          | 0/1 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 1/1 [00:00<00:00, 1849.34it/s]  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 27869.13it/s]
  0%|          | 0/1 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 1/1 [00:00<00:00, 1317.72it/s]Traceback (most recent call last):
  File "shenji_main.py", line 36, in <module>
    main()
  File "shenji_main.py", line 32, in main
    app.listen(8085)
  File "/home/user/anaconda3/envs/syr_torch/lib/python3.7/site-packages/tornado/web.py", line 2140, in listen
    reuse_port=reuse_port,
  File "/home/user/anaconda3/envs/syr_torch/lib/python3.7/site-packages/tornado/tcpserver.py", line 189, in listen
    reuse_port=reuse_port,
  File "/home/user/anaconda3/envs/syr_torch/lib/python3.7/site-packages/tornado/netutil.py", line 162, in bind_sockets
    sock.bind(sockaddr)
OSError: [Errno 98] Address already in use

  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 70492.50it/s]
  0%|          | 0/1 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 1/1 [00:00<00:00, 1971.01it/s]
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 72315.59it/s]
  0%|          | 0/1 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 1/1 [00:00<00:00, 2073.31it/s]
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 60349.70it/s]
  0%|          | 0/1 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 1/1 [00:00<00:00, 1901.32it/s]
  0%|          | 0/2 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 2/2 [00:00<00:00, 67650.06it/s]
  0%|          | 0/1 [00:00<?, ?it/s]100%|¨€¨€¨€¨€¨€¨€¨€¨€¨€¨€| 1/1 [00:00<00:00, 1359.58it/s]